### DISCUSSION TITLE
Fast Food Chain Map

### DISCUSSION THREAD

**Discussion by TheAdonalsium** [jakobzhao/geog495#62] [Final Project]
![image](https://user-images.githubusercontent.com/22570195/145531552-a0d7229d-d132-4c78-a4bb-2f6decd22e85.png)
URL to Repository: https://github.com/Adam-Nicewarner/FastFoodChainMap
URL to Application: https://adam-nicewarner.github.io/FastFoodChainMap/heatmap.html
Map of randomly selected fast food chains. Use the legend to the left to filter for what you want to look for.

**Comment by LiamGearyUW** [comment]
A really solid map that conveys its info well with a great deal of interactivity. Very effective. I'm sure you're aware, however, it's incredibly slow and eats RAM like mad. If I had to guess, I think this is because of the abundance of pngs. I'd recommend using <span> tags where you could, but if that's too much work to switch everything at this point I'd understand. Aside from that, I'd decrease the width of the sidebar.

**Comment by ghost** [comment]
I really like this take on the project, it's very unique. Your average person doesn't always have time to make dinner after a long day out ;) 

My only critique is the margins on your side panel, it's pretty big and distracting, and overall could be stylized cleaner. But this is a really awesome map functionality wise!

**Comment by alexanderlyu** [comment]
A really nice looking map with interesting and attractively styled data. If there was one thing I would improve, it would be having a darker basemap so that your data can stand out even more.

**Comment by lucretiachen99** [comment]
I like the idea of mapping out fast food chains, it's cool to visualize where they are on a larger scale. One improvement you can make is to make the side bar slightly smaller so that the user can see more of the map. It would help to create a search bar as well since there are so many locations.

**Comment by dldl1901s** [comment]
This map is amazing. It's really useful and beneficial. Design is also good because it is intuitively easy to understand. However, it would have been nice if the base map had a lower tone, emphasizing the points more on the map.

**Comment by mhan799** [comment]
Hi Adam,
I really like your idea of this map. I think it will also be very useful in studying the fast-food culture and the fast-food economy of the U.S. I think it would be better in visual if your use layers to show which company dominates in number in each state and use a pop-up window to show the details. Burt overall, this is great already!

**Comment by Anemnox** [comment]
Hi! I really like the styling of the map and how it is visually represented. My favorite aspect is that the basemap is simple to look at with the colored data points. One thing I noticed was that my computer wasn't really able to explore the map very easily because it lagged out. I think one critique i could give is to create an option to limit the amount of datapoints shown so the map could be used on computers with lower specs.

**Comment by shimizusUW** [comment]
I love the idea of this map, especially as a person who loves fast food. I had a hard time looking at all the different data. This could be due to the large amounts of data. Maybe narrow down the location or different restaurants to help improve this.

**Comment by meleakewubbie** [comment]
Your map is very informative. It's interesting to see how many fast food restaurants are on the east side of the United States. One small fix could be shortening the width of the side panel.
